src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -176,6 +177,15 @@ public void setLastViewedClient(Client client) {
         this.lastViewedClient.set(client);
     }
 
+    //=========== Command History =================================================================
+    @Override
+    public String getPreviousCommand() {
+        return commandHistoryStorage.getPreviousCommand();
+    }
+    @Override
```

Comment:
Code style: consider add new line between line 184 and 185.

Reply:
Updated